edx_proctoring/api.py

```diff
@@ -811,7 +811,7 @@ def get_all_exams_for_course(course_id):
     ..
     ]
     """
-    exams = ProctoredExam.get_all_exams_for_course(course_id)
+    exams = ProctoredExam.get_all_exams_for_course(course_id, active_only)
```

Comment:
I think we have to pass `active_only` as a keyword argument like `active_only= active_only`